### PR TITLE
Signup Flow

### PR DESCRIPTION
**Pull request by Basit-qc**

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"clientKey": "FKx1oF94M0OuoDW0YDyAx6tlelUvR3wm", 
+	"clientSecret": "CdaU65M2wJ7G_HJo3eL_NNA-3IOuAbx0llsW46gGoE7FzTIwnqpGsM57tM2AfoMQ"
+}
```
Comment (saleem-latif) [inline src/config.json:4]: nit: add an empty line at the  end of the file.

File: src/interface.ts [hunk 2]
```diff
@@ -20,4 +20,8 @@ export interface IRepoDiffs {
 export interface IFileToDiff {
 	file_path: string;
 	diff: IDiff;
+}
+
+export interface IAuth0User {
+	email: string;
 }
```
Comment (saleem-latif) [inline src/interface.ts:27]: nit: add an empty line at the end of the file.

**Review by saleem-latif — APPROVED** [review]
I am not much familiar with this code but overall it looks okay.

**Comment by saleem-latif** [comment]
@Basit-qc is there a reason for using `tab` instead of space characters for indentation? tabs reduce readability in github.

**Comment by Basit-qc** [comment]
> @Basit-qc is there a reason for using `tab` instead of space characters for indentation? tabs reduce readability in github.

@saleem-latif I think IDE automatically does that. Tried using both VSCode and WebStorm.

File: src/config.json (hunk 1 above)
Comment (Basit-qc) [reply src/config.json]: Thanks, Done

**Comment by Basit-qc** [comment]
@saleem-latif It has the basics of sign up Flow 
- running a server inside plugin, hadnling auth-redirect to plugin using Auth0
- using Norification popups and buttons to navigate user

Same things we will do in Atom & Intellij plugins.  

Next part will be triggering init process.

**Pull request merged**